### DISCUSSION TITLE
Athens rationale for end of development?

### DISCUSSION THREAD

**Discussion by hkhiro** [athensresearch/athens#2430] [General]
Is there a reason /rationale for the end of development on this most interesting project?

Have been using Logseq but am frustrated with the lack of ability to self host and so have now discovered this offering.

But it's "no longer being maintained" - whyyy?

**Comment by Riteshkrpanda** [comment]
same feeling bro i tried the demo and it felt so smooth compared to logseq i do not know why they decided to completely shut down the project

**Comment by BradKML** [comment]
My bet is that YCombinator is probably doing some weird business manoeuvre, see what ideas work for FOSS PKM, and then inject them back into proprietary systems for profit. If the app is too smooth, paradoxically, it means they better work elsewhere, at least now with AI coding agent, we might be able to resuscitate "dead" projects.
In fact, the same things are happening with coding agents nowadays, and anything that bridges PKMs with automated coding agents would be powerful in the FOSS realm. https://github.com/codestoryai/aide
